publish/pipeline-build-deploy-app/pipeline-build-deploy-app.md

```diff
@@ -0,0 +1,167 @@
+---
+permalink: /guides/pipeline-build-deploy-app/
+layout: guide-markdown
+title: Build and deploy applications with pipelines
+duration: 30 minutes
+releasedate: 2020-05-01
+description: Explore how to use pipelines to build and deploy applications
+tags: ['Pipelines, Application stacks']
+guide-category: pipelines
+---
+
+<!-- Note:
+> This repository contains the guide documentation source. To view
+> the guide in published form, view it on the [website](https://kabanero.io/guides/{projectid}.html).
+-->
+
+<!--
+//
+//	Copyright 2019, 2020 IBM Corporation and others.
+//
+//	Licensed under the Apache License, Version 2.0 (the "License");
+//	you may not use this file except in compliance with the License.
+//	You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+//	Unless required by applicable law or agreed to in writing, software
+//	distributed under the License is distributed on an "AS IS" BASIS,
+//	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//	See the License for the specific language governing permissions and
+//	limitations under the License.
+//
+-->
+
+## Introduction
+
+This guide will walk you through the steps needed to run a pipeline to build your application, publish the image to a registry, and optionally deploy the application on your cluster.
+
+For this guide, we will use the one of the default pipelines, `java-openliberty-build-deploy-pl`, that is activated by the Kabanero operator. You can replace this with any of the other `build-deploy` pipelines or your custom pipeline.
+
+## Steps
+
+1. Clone the `https://github.com/kabanero-io/kabanero-pipelines/` repository.
+   * cd into the `pipelines/sample-helper-files` directory
+2. Make sure you have the appropriate storage necessary to drive pipelines. If you have a dynamic storage provisioner available on your cluster, you can proceed to the next step. If not, you can set up a simple NFS-based persistent volumen (PV) using the nfs-pv.yaml sample provided.
+   * Note: you must update the IP address before applying the file.
+   * Run `oc apply nfs-pv.yaml`
+3. If you are using a private GitHub repo, create the secret for your repo. Follow the steps in Secrets section of [Working with Pipelines](../working-with-pipelines/working-with-pipelines.html#getting-started).
+4. Determine whether it is necessary to create the secret for your image registry.
```

Comment:
I think I would lead into the list slightly differently as I was initially unsure what to do ... i.e. "Read the following information to determine ....."